polaris-react/src/components/ResourceItem/ResourceItem.tsx

```diff
@@ -156,6 +160,7 @@ class BaseResourceItem extends Component<CombinedProps, State> {
       verticalAlignment,
       dataHref,
       breakpoints,
+      onMouseOver = () => {},
```

Comment:
I don't think we need a default value for this.

```suggestion
      onMouseOver,
```